nms/app/packages/magmalte/app/components/context/FEGSubscriberContext.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * Copyright 2020 The Magma Authors.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * @flow strict-local
+ * @format
+ */
+
+import type {network_id, subscriber_state} from '@fbcnms/magma-api';
+
+import React from 'react';
+
+export type FEGSubscriberContextType = {
+  sessionState: {[network_id]: {[string]: subscriber_state}},
```

Comment:
Rename this please - it's confusing in other files that this is keyed on network id